src/zap/zapinfo.cpp

```diff
@@ -4497,6 +4497,12 @@ bool ZapInfo::canTailCall(CORINFO_METHOD_HANDLE caller,
                                          CORINFO_METHOD_HANDLE exactCallee,
                                          bool fIsTailPrefix)
 {
+#ifdef FEATURE_READYTORUN_COMPILER
+    // READYTORUN: FUTURE: Delay load fixups for tailcalls
+    if (IsReadyToRunCompilation())
+        return false;
```

Comment:
CoreRT is only using CoreCLR RyuJIT. It does not use CoreCLR JIT-EE interface implementation. The CoreRT IL compiler has own implementation of the JIT-EE interface (https://github.com/dotnet/corert/blob/01cdef0f099e507230e4bfa26e576dfd62a61306/src/JitInterface/src/CorInfoImpl.cs#L507).